en/mulesoft/modules/ROOT/pages/ms_composer_build_a_flow_using_templates.adoc

```diff
@@ -13,4 +13,20 @@ Some examples of templates include:
 
 For more information about how to create a new flow from template, refer to xref:ms_composer_flows.adoc#create-a-new-flow-from-template[Create a New Flow From Template].
 
+== Customizing Your Template
+
+Since you will likely need to further customize your flow from a template, consider the following recommendations and guidelines when encountering messages while customizing your flow from template.
```

Comment:
```suggestion
Since you will likely need to further customize your flow from a template, consider the following recommendations and guidelines when encountering messages while customizing your flow from a template.
```

Reply:
This sentence is a bit repetitive. Consider removing the first part of it and starting with "Consider..."